scripts/store_organization_ipfs.py

```
@@ -0,0 +1,60 @@
+    """
+    Helper script to create a market on Omen, usage:
+
+    ```bash
+    python scripts/store_prediction.py \
+        --from-private-key your-private-key
+        --organization_name my-organization
+
+    ```
+    """
+
+    assert (len(organization_name) < 32, "Organization name too long")
```

Ruff (0.11.9)

Assert test is a non-empty tuple, which is always `True`

(F631)